.github/workflows/atd_moped_database.yml

```diff
@@ -0,0 +1,50 @@
+#
+# Builds Cognito Triggers for Moped Authentication
+#
```

Comment:
can this be updated to reflect what the script does?

Reply:
yes thank you ✅